README.md

```diff
@@ -705,7 +705,7 @@ be called when you want, even after some data was inserted, though it's best to
 call it at application startup. The options are:
 
 * **fieldName** (required): name of the field to index. Use the dot notation to
-  index a field in a nested document.
```

Comment:
I would appreciate a modification to the typings file with the corresponding changes.

Reply:
done